app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -224,6 +270,23 @@ public int getTotalMediaCount() {
     return adapter.getItemCount();
   }
 
+  /**
+   * Open the editor for the language Wikipedia
+   *
+   * @param contribution
+   */
+  @Override
+  public void onConfirmClicked(@Nullable Contribution contribution, boolean copyWikicode) {
+    if(copyWikicode) {
+      String wikicode = contribution.getWikiCode();
+      Utils.copy("wikicode", wikicode, getContext());
+    }
+
+    final String url = languageWikipediaSite.mobileUrl() + "/wiki/" + contribution.getWikidataPlace()
+        .getWikipediaPageTitle() + "#/editor/0";
+    Utils.handleWebUrl(getContext(), Uri.parse(url));
```

Comment:
I think we agree that the user shouldn't be directed to the editor anymore. So, we can stop appending `#/editor/0`.

Reply:
Updated.